modules/auxiliary/scanner/ssh/ssh_login.rb

```diff
@@ -92,6 +114,35 @@ def run_host(ip)
     @ip = ip
     print_brute :ip => ip, :msg => 'Starting bruteforce'
 
+    if datastore['USER_FILE'].blank? && datastore['USERNAME'].blank? && datastore['USERPASS_FILE'].blank?
```

Comment:
If you want to avoid chaining `&&` maybe something like?
```suggestion
    if datastore.values_at('USER_FILE', 'USERNAME', 'USERPASS_FILE').all?(&:blank?)
```